### DISCUSSION TITLE
bugreport от loki orc

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2_classic#85] [General]
https://docs.google.com/document/d/1wfbN-DngJe0o-bhvStwquOd-28ifvJb5PQQBDcyLVaQ/edit

**Comment by Kor-Angar** [comment]
### Диалоги

- Озвучка ГГ в 1-м диалоге с Бастером про колонию (для поиска по Ctrl+F) отличается от текста, что-то вроде “Да, так и было”, когда Бастер его узнает.
- Диалог с Ренгару при поимке старый, классический, в котором ничего не понятно, в аддоне в сто раз лучше. Я из-за на него каждый раз косячу начало ГВ
- Не озвучена вовсе, или не работает только в пещере под мостом озвучка диалога “мне никогда не открыть это без правильного ключа”
- Если сначала побить Феллана, а потом взять квест у Алвина, а потом подойти к Феллану, он скажет что больше не будет, НО первый диалог “не мог бы перестать колотить и его ответы, что нет” останется
- у Бенгара среди крестьян - фермер
- Диалог с Фестером про подготовку к бою. Разница в тексте и озвучке ГГ -  Почему/Зачем

**Comment by Kor-Angar** [comment]
какие то проблемы с кавысками


нв |   классик  
-- | --
 ![image](https://user-images.githubusercontent.com/49514764/143091556-72c508ec-98be-4a67-a018-c449bd37d2a7.png)  |  ![image](https://user-images.githubusercontent.com/49514764/143091722-257d726c-74fe-4fec-823a-02684c84ec78.png)

**Comment by dosinabox** [comment]
> https://docs.google.com/document/d/1wfbN-DngJe0o-bhvStwquOd-28ifvJb5PQQBDcyLVaQ/edit

В основном исправлено.
